index.html

```diff
@@ -0,0 +1,136 @@
+                <ul class="nav-items">
+                    <li class="nav-item"><a href="./about.html" class="nav-link">About</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link camp">AI Campaign</a></li>
+                </ul>
+            </div>
+            <div class="mobile-menu">
+                <ul class="mobile-nav-items">
+                    <li class="nav-item"><a href="./index.html" class="nav-link" id="home-link"></a></li>
+                    <li class="nav-item"><a href="./about.html" class="nav-link" id="about-link">About</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link" id="program-link">Program</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link" id="join-link">Join</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link" id="sponsor-link">Sponsor</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link" id="news-link">News</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link" id="campaign-link"></a></li>
+                </ul>
+            </div>
+        </nav>    
+    </header>
+
+    <section class="headline-section" id="headline">
+        <h1 class="main-headline">
+            <span class="span-headline-one">COME! SEE THE FUTURE.</span>
+            <br>
+            <span class="span-headline-two">AFRICA ARTIFICIAL INTELLIGENCE EXPOSITION</span>
+            <br>
+            <span class="span-headline-two">2022</span>
+        </h1>
+        <div class="headline-description-container">
+            <p class="headline-description">Africa AI Expo aims at showcasing some of the artificial intelligence projects in the continent 
+                that positively impact people's lives and improve the continents competitiveness in the world market market.</p>
+        </div>
+        <div class="event-details">
+            <h3 class="date">2022.08.05(Wed) ~ 07(Fri)</h3>
+            <h5 class="location">@ ICT Center Of Excellence, Ndola, Zambia.</h5>
+        </div>
+    </section>
+
+    <section class="program-section" id="program">
+        <h2 class="program-heading">Main Program</h2>
+        <hr class="program-line">
+        <div class="program-container">
+            <div class="program-item">
+                <img src="./images/program1.png" alt="Lecture" class="program-icon">
+                <h4 class="title">Speak</h4>
+                <p class="program-description">Lorem ipsum dolor sit amet consectetur.</p>
+            </div>
```

Comment:
- [ ] To make the code more semantic, I would suggest making the `div` wrappers with the program-item class changed to `article` tags

```
<article class="program-item"></article>
```

Reply:
Hi @Cathella, Thanks for the review. 
I have fixed the issues above. Kindly review them before I submit another request for review.
Regards.